src/components/bottom_bar/bottom_bar.ts

```diff
@@ -152,6 +152,7 @@ export class BottomBar extends Component<Props, SpreadsheetChildEnv> {
         textColor: sheet.isVisible ? undefined : "grey",
         execute: (env) => {
           env.model.dispatch("ACTIVATE_SHEET", { sheetIdFrom: from, sheetIdTo: sheetId });
+          this.scrollToSheet(sheetId);
```

Comment:
Can't you use [scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) to scroll instead of having to manually compute the scroll ? Or does it not work ?

Reply:
Ah, you're right! I wasn't aware that such a straightforward solution exists. This is much simpler and exactly what we need here. However, we will need to mock it in order to test the behavior.